be/src/storage/tablet_meta.h

```diff
@@ -261,6 +267,8 @@ class TabletMeta {
     //    a rowset will not be removed from _inc_rs_metas if only the binlog is expired
     BinlogLsn _binlog_min_lsn;
 
+    bool _disable_shortcut_compaction = false;
```

Comment:
An active name is better. `enable_short_compaction`

Reply:
done